src/packages/solid/components/Checkbox/Checkbox.tsx

```diff
@@ -30,10 +30,18 @@ export interface CheckboxProps extends NodeProps {
 const Checkbox: Component<CheckboxProps> = (props: CheckboxProps) => {
   return (
     <View
+      forwardStates
       {...props}
+      tone={props.tone ?? styles.tone}
+      style={props?.style?.Container ?? styles.Container}
```

Comment:
Here we can start doing this now:
={[props.style?.Container, styles.Container]}

Reply:
we should be able to include the below tone setup in that array as well, like we were saying in Icon